### DISCUSSION TITLE
Net 6 and Net 7

### DISCUSSION THREAD

**Discussion by FDE-LC2L** [Kinnara/ModernWpf#513] [General]
Hello. For development in the process of migrating to Net7 I adapted ModernWPF so that it compiles and works with Net7 (and, in theory, with Net6 ;-) ).
I don't want to fork or commit here because my work isn't necessarily very clean. There are still a lot of warnings, especially about nullables. With my work ModernWPF compiles in net 7, "ModernWpf.SampleApp" app compiles and works and my app compiles and works :-)
However, as ModernWPF has helped me in the past, if anyone is interested in my code, you can download it here https://www.lc2l-solutions.fr/download/modernwpf/ModernWPF%20Net6-7.7z
If ModernWPF developers want to use my code that's fine, they can. If other people want to use my code they can. I would be happy if my code can help one or two developers.
Have à nice day.

**Comment by maksimwest00** [comment]
@FDE-LC2L its rly good! can you learn me how you do it for experience pls ??@sharyyla my telegram
